tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeSpacetimeQuantities.cpp

```diff
@@ -179,6 +180,20 @@ void test_compute_spatial_metric_lapse_shift(const T& used_for_size) {
   CHECK_ITERABLE_APPROX(lapse, lapse_test);
 }
 
+template <size_t Dim, typename DataType>
```

Comment:
You should test this in its own `Test_DerivativeSpatial...` file since this file is rather unwieldy already.